tests/cursor/cursor-getmore-005.phpt

```diff
@@ -1,15 +1,22 @@
 --TEST--
 MongoDB\Driver\Cursor query result iteration with getmore failure
---XFAIL--
-START() tests must be reimplemented (PHPC-1179)
 --SKIPIF--
 <?php require __DIR__ . "/" ."../utils/basic-skipif.inc"; ?>
-<?php START("THROWAWAY", ["version" => "30-release"]); CLEANUP(THROWAWAY); ?>
+<?php
+/* This test spins up its own mongod instance, so only run this in the most default "standalone, no
+ * auth" configurations. This way, we can test on multiple server versions, but not waste resources
+ * on f.e. Travis. */
```

Comment:
Random thought, but is this the same approach you have for testing Atlas connections?

I'm curious if we should simply have a separate matrix entry that essentially implies "the test will use its own environment", so that these "spin up our own MO environment" and Atlas tests run in an isolated matrix entry.